Xamarin.Forms.Platform.Android/AppCompat/ImageButtonRenderer.cs

```diff
@@ -172,10 +172,15 @@ void IVisualElementRenderer.SetElement(VisualElement element)
 			UpdateInputTransparent();
 			UpdatePadding();
 
-			ElementChanged?.Invoke(this, new VisualElementChangedEventArgs(oldElement, ImageButton));
+			OnElementChanged(new ElementChangedEventArgs<ImageButton>(oldElement, ImageButton));
 			ImageButton?.SendViewInitialized(Control);
 		}
 
+		protected virtual void OnElementChanged(ElementChangedEventArgs<ImageButton> e)
```

Comment:
it's better to keep the `virtual void OnElementChanged` empty and invokes it after firing the event

Reply:
I mainly did it like this because this is how VisualElementRenderer does it
https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/VisualElementRenderer.cs#L305

So I was just trying to mimic that one for the sake of consistency

Reply:
For the record I do agree with @StephaneDelcroix 

But  I did it this way because this is the same way we do it literally everywhere that I could find. So I feel like at this point consistency should be the winner

https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.iOS/Renderers/WkWebViewRenderer.cs#L126

Reply:
Yeah, agreed. Consistency is best. :/

Reply:
> Yeah, agreed. Consistency is best. :/

disagreed, but I'm not going to die on this hill